Source/ConvolutionFunctions/arm_transpose_conv_s8.c

```diff
@@ -172,11 +172,28 @@ arm_cmsis_nn_status arm_transpose_conv_s8(const cmsis_nn_context *ctx,
                 }
             }
         }
-
         img_data = img_buf_ptr;
         for (int i = 0; i < output_x * output_y; i++)
         {
-            for (int i_output_ch = 0; i_output_ch < output_ch; i_output_ch++)
+            int i_output_ch = 0;
+#if defined(ARM_MATH_MVEI)
+            int8_t *ip_out_data = output_data_ptr;
+            for (int32_t num_channels = output_ch; num_channels > 0; num_channels -= 4, i_output_ch += 4)
```

Comment:
could you use the same naming convention for the loop variables like the main loops above?
so something like i_output_ch = output_ch and so on. Similarly, a non i_ name for the existing i_output_ch so it is different from the loop varaible.

or you could make i_output_ch to be the control variable for the loop and move, num_channels outside of the loop..

Reply:
done :)